docs/includes/generated_docs/schemas/beta.tpp.md

```diff
@@ -223,7 +223,11 @@ return ordered_addrs.last_for_patient()
 <p class="dimension-indicator"><code>many rows per patient</code></p>
 ## apcs
 
+Admitted patient care spells (APCS) data is provided via the NHS Secondary Uses Service.
 
+This table gives core details of spells.
+
+Each row is an in-hospital spell: a period of continuous care within a single trust.
```

Comment:
comment: This is from the current [OpenSAFELY docs](https://docs.opensafely.org/data-sources/apc/).